jobs/aws_cpi/spec

```diff
@@ -20,6 +20,9 @@ properties:
   aws.secret_access_key:
     description: AWS secret_access_key for the aws cpi (Required when aws.credentials_source is set to `static`)
     default: null
+  aws.session_token:
```

Comment:
@keymon Does that mean a user can only set the session token and no access key or secret key?

Reply:
No. You can either specify:
 - secret_access_key + access_key_id: Normal static credentials.
 - secret_access_key + access_key_id + session_token: STS credentials.

In the first case the `session_token` would be null.

Reply:
@keymon Sounds good!